layouts/partials/comment.html

```diff
@@ -124,6 +124,43 @@
                 Please enable JavaScript to view the comments powered by <a href="https://utteranc.es/">Utterances</a>.
             </noscript>
         {{- end -}}
+
+        {{- /* Waline Comment System */ -}}
+        {{- $waline := $comment.waline | default dict -}}
+        {{- if $waline.enable -}}
+            <div id="waline"></div>
+			<script src='{{ $waline.js }}'></script>
+			<script>
+			Waline({
+			  el: '#waline',
+			  serverURL: '{{ $waline.serverURL }}',
+			  visitor: '{{ $waline.visitor }}',
+			  dark: '{{ $waline.dark }}',
+			});
+			</script>
+            <noscript>
+                Please enable JavaScript to view the comments powered by <a href="https://waline.js.org/">Waline</a>.
+            </noscript>
+        {{- end -}}
+
+        {{- /* Twikoo Comment System */ -}}
+        {{- $twikoo := $comment.twikoo | default dict -}}
+        {{- if $twikoo.enable -}}
+			<div id="twikoo"></div>
+			<script src="{{ $twikoo.js }}"></script>
```

Comment:
Can you use something like this?

https://github.com/sunt-programator/CodeIT/blob/a74cdcf1dfd2e1ceb0a997ae3a045c8b088abe24/layouts/partials/comment.html#L19-L31

We have the script partial page for this, and we can use also the file from the `jsdelivr.yml` file. Also, as a default value, it would be nice to have a local copy of the twikoo/waline scripts files. They are stored in this location: `assets/lib`.

Reply:
I tried...but I only learned javascript for a few weeks...I hope you can help adjust if necessary